locales/languages/en.json

```
@@ -477,7 +477,7 @@
       "state": "State/Region",
       "city": "City",
       "postal_code": "Postal/Zip Code",
-      "country": "Country"
+      "country": "Country",
```

ah, JSON do not support trailing commans
```suggestion
      "country": "Country"
```